brick/brick.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	fmt.Println("Enter the hole dimensions (2 numbers):")
+	var x, y float64
+	fmt.Scan(&x, &y)
+	fmt.Print("Enter the brick dimensions (3 numbers):")
+	var a, b, c float64
+	fmt.Scan(&a, &b, &c)
+	if x >= a && y >= b {
```

Comment:
Could you think about a solution that avoids a lot of similar `if`s? Perhaps a solution that uses sorting.